Example/Tests/Rules/Inline/ShortLinkRuleTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ShortLinkRuleTests.swift
+//  markymark_Tests
+//
+//  Created by Jim van Zummeren on 20/07/2021.
+//  Copyright © 2021 M2mobi. All rights reserved.
+//
+
+import XCTest
+@testable import markymark
+
+class ShortLinkRuleTests: XCTestCase {
+
+    var sut: ShortLinkRule!
+
+    override func setUp() {
+        super.setUp()
+        sut = ShortLinkRule()
+    }
+
+    func test_WillRecognizesLines_When_CallingRecognizesLinesWithValidInput() {
+        XCTAssertTrue(sut.recognizesLines(["<https://www.m2mobi.com>"]))
+        XCTAssertTrue(sut.recognizesLines(["<http://www.m2mobi.nl/something/other?id=4&page=4>"]))
+        XCTAssertTrue(sut.recognizesLines(["<hkia://flights>"]))
+        XCTAssertTrue(sut.recognizesLines(["<google.com>"]))
+    }
+
+    func test_WillNotRecognizesLines_When_CallingRecognizesLinesWithInvalidInput() {
+        XCTAssertFalse(sut.recognizesLines(["<Google>"]))
+        XCTAssertFalse(sut.recognizesLines(["<$$>"]))
+        XCTAssertFalse(sut.recognizesLines(["<hello>"]))
```

Comment:
is it possible to test for "<>" ? I believe the regex also supports empty elements for this rule.